kivy/core/window/window_sdl2.py

```diff
@@ -714,7 +714,8 @@ def request_keyboard(self, callback, target, input_type='text'):
 
     def release_keyboard(self, *largs):
         super(WindowSDL, self).release_keyboard(*largs)
-        self._win.hide_keyboard()
+        if self._system_keyboard.widget:
+            self._win.hide_keyboard()
         self._sdl_keyboard = None
```

Comment:
@matham i have used kivy apps without that for weeks now and didn't see any bad effects, is there a good reason for disabling textinput events?